client/control/sass/control.scss

```diff
@@ -382,3 +382,12 @@
       white-space: nowrap;
     }
 }
+
+/* Recipe List */
+#recipe-list {
```

Comment:
I agree wrt using id's for selectors - I was just following the conventions already established in `control.scss`. I'll switch to a class selector instead.